### ISSUE TITLE
Shared place icon

### ISSUE THREAD

**Issue by HerzScheisse** [vesta-webtrees-2-custom-modules/vesta_shared_places#182]
Hi,

while searching for a replacement for the icon in front of the shared place location I noticed that in the folder
`/webtrees/modules_v4/vesta_shared_places/resources/css/webtrees/plac.png`
seems to be not used at any place (did look up the code... 

Am I right that this is maybe a leftover and the icon is generated from css code in `webtrees.css` and so the icon itself can be deleted?!

**Comment by HerzScheisse** [comment]
and my second question regarding this topic...

do you know a way how to update your code with this icon, so that other styles can use a different icon... like if some is using the github-webtrees-theme...

the changes in that theme module css does not have any effect... so for now i changed the shared places webtrees.css to:

```
.wt-icon-shared-place {
  --content: "\f387";
}
```

**Comment by ric2016** [comment]
Yes, plac.png is just left there as internal reference and functionally could be removed.

> do you know a way how to update your code with this icon, so that other styles can use a different icon

Create a separate <theme>.css file in that folder. I don't think there is a better way currently.
